runtime/compiler/env/j9method.cpp

```diff
@@ -6757,6 +6757,39 @@ TR_ResolvedJ9Method::getResolvedDynamicMethod(TR::Compilation * comp, I_32 callS
 #else
    TR_ResolvedMethod *result = NULL;
 
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+   TR_ResolvedMethod * owningMethod = comp->getCurrentMethod();
```

Comment:
`owningMethod` should be `this`. We're looking at invokedynamic bytecode of `this` resolved method.